cdap-watchdog-api/src/main/java/io/cdap/cdap/api/metrics/MetricValue.java

```diff
@@ -24,12 +24,64 @@
   private MetricType type;
   private long value;
 
+  // following fields to support distribution aka histogram/event metrics
+  /*
+  Based on https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TypedValue#Exponential
```

Comment:
We should not refer google cloud in the generic  API. Please put a short description in the javadoc.

Reply:
Removed google doc references.